signal_scanner_bot/twitter.py

```diff
@@ -35,16 +36,73 @@ def get_api():
 ################################################################################
 def send_tweet(tweet: str, timestamp: datetime, api: tweepy.API) -> None:
     hashtags = " ".join(SEND_HASHTAGS)
-    if len(tweet + hashtags) >= 260:
-        # TODO: better
-        log.warning(f"Cannot tweet message, exceeds length: {tweet}")
-        return
-
-    formatted = dedent(
-        f"""
-    {tweet} @ {timestamp.strftime('%l:%M:%S%p').strip()}
-
-    {hashtags}
-    """
-    )
-    api.update_status(formatted)
+
+    # Check if tweet is longer than 280 minus a 20 char padding, add to list if
+    # not, continue with dividing up text if so
+    if len(tweet + hashtags) >= TWEET_MAX_SIZE - TWEET_PADDING:
+
+        # Split tweet into word list, create empty list to store serialized tweets, index to track last position
+        tweet_word_list = tweet.split(" ")
+        tweets_list = []
+        j = 0
+
+        # For loop through word list and enumerate the index because we'll need
+        # it but don't actually care about the list value so send to null
+        for index, _ in enumerate(tweet_word_list):
```

Comment:
For this (and below on line 74) you can actually use the following logic without having to extract values from the list:
```suggestion
        for index in range(len(tweet_word_list)):
```